### ISSUE TITLE
Add Health Check(s) to @amplication/server

### ISSUE THREAD

**Issue by timothy-amplication** [amplication/amplication#2192]
It's good practice to include a health check with your services. This is specifically important when deploying to systems like Kubernetes where it can handle traffic routing based on healthy endpoints/services. 

NestJS has this already figured out and provides a set of [Health Checks](https://docs.nestjs.com/recipes/terminus#healthchecks-terminus) available.

Some health-checks `@amplication/server` could benefit from is...
1. `HttpHealthIndicator` - A basic HTTP endpoint to report that server that is listening on a port is running and accepting requests.
2. `TypeOrmHealthIndicator` / `SequelizeHealthIndicator` - Reports that the server is connected to external database(s) - Useful in case the server can no longer connect to the DB and Kubernetes can stop routing traffic to the service.
3. `DiskHealthIndicator` - Reports when Disk Space for a specific volume/mount is low - Could be useful for Amplication - maybe not?

**Comment by mshidlov** [comment]
@timothy-amplication, this looks great. I added it to Amplication board and milestone `0.11.2`.

As for the `HttpHealthIndicator` in the docs, it looks like it performs health checks on an external endpoint,
and for the DB connection, we are using Prisma and not TypeORM\Sequelize.


A basic HTTP endpoint to report that the server is listening on a port, running, and accepting requests is what we need. So no additional 3rd party library is required, just an endpoint that returns HTTP 204 (empty response). 

We added to the generated app a few weeks ago exactly those endpoints.
please look on it [here](https://github.com/amplication/amplication/pull/1778/files).

What do you think will be a good solution for it?

**Comment by timothy-amplication** [comment]
@mshidlov Let me do a little bit more research on this topic and get back to you on what I'd think a good solution is. I'll most likely scope this out after https://github.com/amplication/amplication/issues/2165 and https://github.com/amplication/amplication/issues/2164 are completed.

**Comment by yuval-hazaz** [comment]
@mshidlov 
is this still open?
would you like to move forward with the approach we implemented in the generated app?

**Comment by mshidlov** [comment]
@yuval-hazaz yes this is still open and yes we can continue with the approach we implemented in the generated app

**Comment by rizdev99** [comment]
Hey, @souravjain540 
Can you please assign this issue to me? 
PS - I am a newbie to open source :)

**Comment by souravjain540** [comment]
Sure! @rizdev99.

**Comment by souravjain540** [comment]
@rizdev99 are you working on it?

**Comment by rizdev99** [comment]
yes @souravjain540 , I'm still working on it. I am not able to find my way through it. If someone else is up for this then you can unassign me from this issue. I might need to level up my skills

**Comment by rizdev99** [comment]
thanks @yuval-hazaz 
I know it's been a lot of time for this small issue. I have joined the discord channel so I'll be looking over there for help. Also, I'll try to commit the changes by this weekend, If not I am not able to do so then I'll unassign myself and will pass this opportunity to someone else

**Comment by rizdev99** [comment]
i unassigned myself as im not able to proceed thank you

**Comment by fosa-ij** [comment]
Hi, good day. 
so I'm looking to contribute to this project but need to know the exact skills required for this project, as I'm not so sure.

**Comment by nya1** [comment]
> Hi, good day. so I'm looking to contribute to this project but need to know the exact skills required for this project, as I'm not so sure.

Hello, you can checkout the [contribution](https://github.com/amplication/amplication/blob/master/CONTRIBUTING.md) file to learn more about the tech stack used by Amplication, I think a knowledge of NestJS and Typescript is needed.
If the issue is still up for grabs I would gladly work on it.

**Comment by fosa-ij** [comment]
@nya1 okay, thanks a lot I appreciate.
Seems I might have to look elsewhere as I have no knowledge of either skill
I'm focused on MERN stack.
